SYS_ATL/LoopIR_scheduling.py

```diff
@@ -2657,6 +2744,9 @@ def map_s(self, s):
         return super().map_s(s)
 
 
+# Note: This analysis is overly conservative.
+# However, it might be a bit involved to come up with
+# a more precise analysis.  TODO
```

Comment:
If you start a comment with `# TODO:` it is easier to automatically track (e.g. with `grep`).